### PR TITLE
Fewd 9785 3.2 beta1 release notes

### PR DESCRIPTION
**Pull request by ElliotFrancisHunter**
Relnotes for Android and iOS 3.2 Beta 1.

Also removed todos for other relnotes and left them as 'none for this release' if people open them.

Added new section specific to the vector search extension in the release notes

### CODE REVIEW

File: modules/c/partials/release-notes/couchbase-mobile-c-release-note.3.2.0.adoc [hunk 1]
```diff
@@ -1,25 +1,42 @@
 [#maint-3-2-0]
-== 3.2.0 -- TODO 2024
+== 3.2.0 Beta 1 -- March 2024
```
Comment (osfameron) [inline modules/c/partials/release-notes/couchbase-mobile-c-release-note.3.2.0.adoc:2]: I think it sounds like we should NOT have this section, and instead at most a "IMPORTANT: 3.2.0 is not released for this platform" banner instead?

File: modules/csharp/partials/release-notes/couchbase-mobile-csharp-release-note.3.2.0.adoc [hunk 2]
```diff
@@ -1,25 +1,42 @@
 [#maint-3-2-0]
-== 3.2.0 -- TODO 2024
```
Comment (osfameron) [inline modules/csharp/partials/release-notes/couchbase-mobile-csharp-release-note.3.2.0.adoc:2]: Yeah, sorry, think we need to remove these sections, as there is no 3.2 release for these platforms

**Review by osfameron — CHANGES_REQUESTED** [review]
Need to remove from the platfroms that aren't getting it

File: modules/c/partials/release-notes/couchbase-mobile-c-release-note.3.2.0.adoc (hunk 1 above)
Comment (ElliotFrancisHunter) [reply modules/c/partials/release-notes/couchbase-mobile-c-release-note.3.2.0.adoc:2]: IMPORTANT admonition is a good shout

**Review by osfameron — APPROVED** [review]
Better - good to go for now, though I'd remove the 3.2.0 relnote from page entirely, perhaps as 1st iteration after GA

**Comment by ElliotFrancisHunter** [comment]
This is fine for now but requires tidying

**Pull request merged**